### PR TITLE
Deprecate dotnet 3 x

### PR DESCRIPTION
**Pull request by brayanhenao**
- Changes to `fixtures` and `init_test.go`
- Update all functions marked as `deprecated` by go.

### CODE REVIEW

File: src/dotnetcore/integration/sealights_test.go [hunk 1]
```diff
@@ -1,59 +1,59 @@
-package integration_test
```
Comment (arjun024) [inline src/dotnetcore/integration/sealights_test.go:1]: @brayanhenao What's the diff in this file? I can't figure out.
Same for `src/dotnetcore/hooks/sealights.go`

Comment (brayanhenao) [reply src/dotnetcore/integration/sealights_test.go:1]: @arjun024 That's weird, maybe LF format?

Comment (arjun024) [reply src/dotnetcore/integration/sealights_test.go:1]: Yes `git show --ignore-space-at-eol -- src/dotnetcore/integration/sealights_test.go` is empty.
Can you unstage both of those files?

Comment (brayanhenao) [reply src/dotnetcore/integration/sealights_test.go]: @arjun024 Done!

**Review by arjun024 — APPROVED** [review]

**Pull request merged**